script/runtests/runtests.py

```diff
@@ -0,0 +1,331 @@
+#####
+# Run this script from qa-infra/script/runtests.
+# e.g.: python3 runtests.py -l log.log -o reactors_options.json
+#
+# It will replace the Makefile and testnet.toml files in qa-infra, that is, ../../.
+# Do not commit the updated files!!!
+#
+# If the Makefile and testnet.toml files are updated, the templates in this folder should
+# also be updated.
+####
+
+
+from __future__ import print_function
+
+from __future__ import absolute_import
+
+import os
+import sys
+import time
+from datetime import datetime
+import getopt
```

Comment:
Why not just use [argparse](https://docs.python.org/3/library/argparse.html)? Even in getopt's docs they recommend to use argparse instead.